roles/install_openstack_ca/tasks/main.yml

```diff
@@ -49,7 +56,7 @@
     get_checksum: false
     get_mime: false
 
-- name: Inject OpenStackControlplane CA bundle  # noqa: no-handler
+- name: Inject OpenStackControlplane CA bundle # noqa: no-handler
```

Comment:
I'd even nest that under a block:
```YAML
- name: manage CA if available
  when:
    - ca_bundle_data.rc == 0
  block:
    - name:  Creating tls-ca-bundle.pem
      ....
    - name: Inject OpenStackControlplane CA bundle
      ....
```

WDYT?

Reply:
Yeah I see what you mean, let me put that in a new PR now because I don't think it's 100% relevent to this one.

Reply:
https://github.com/openstack-k8s-operators/ci-framework/pull/2203